src/solvers/dgsem_tree/containers_2d.jl

```diff
@@ -1372,7 +1372,7 @@ function Base.resize!(container::ContainerSubcellLimiterIDP2D, capacity)
     @unpack _alpha, _alpha1, _alpha2 = container
     resize!(_alpha, n_nodes * n_nodes * capacity)
     container.alpha = unsafe_wrap(Array, pointer(_alpha), (n_nodes, n_nodes, capacity))
-    container.alpha .= zero(eltype(container.alpha))
+    container.alpha .= -1 * one(eltype(container.alpha))
```

Comment:
```suggestion
    container.alpha .= 0 / zero(eltype(container.alpha))
```
This will create `NaN` for `Float64` and `NaN32` for `Float32`. I don't know if there is a more canonical way to achieve this.

Reply:
Maybe
```suggestion
    container.alpha .= convert(eltype(container.alpha)), NaN)
```
?

Reply:
Wouldn't work for Float32 GPUs, or would it?

Reply:
```julia
julia> convert(Float32, NaN)
NaN32
```

Reply:
No, I mean that since `NaN` is a value of type `Float64`, it would already be evaluated as such before being passed into `convert`. But I think in practice, the call to `convert` is going be optimized away anways, so let's follow YAGNI and use the simplest yet generic approach as suggested [above](https://github.com/trixi-framework/Trixi.jl/pull/1611#discussion_r1342327584).